### ISSUE TITLE
Input placeholder color contrast does not meet WCAG 2.0 AA requirements

### ISSUE THREAD

**Issue by kgibb** [carbon-design-system/carbon#7515]
## Detailed description

> What version of the Carbon Design System are you using?

carbon-components@10.14.0

> What did you expect to happen?

Placeholder contrast ratio should be at least 4.5:1.

> What WCAG 2.1 checkpoint does the issue violate?

1.4.3 Contrast Minimum Level AA

## Placeholder Contrast Ratios

| Theme | Light | Background | Foreground | Ratio |
|--------|------|-------------|-------------|-------|
| white | :heavy_check_mark: | #ffffff | #a8a8a8 | 2.37:1 |
| white | :x: | #f4f4f4 | #a8a8a8 | 2.16:1 |
| g10 | :heavy_check_mark: | #f4f4f4 | #a8a8a8 | 2.16:1 |
| g10 | :x: | #ffffff | #a8a8a8 | 2.37:1 |
| g90 | :heavy_check_mark: | #525252| #6f6f6f | 1.55:1 |
| g90 | :x: | #393939 | #6f6f6f  | 2.29:1 |
| g100 | :heavy_check_mark: | #393939 | #6f6f6f  | 2.29:1 |
| g100 | :x: | #262626 | #6f6f6f  | 3.01:1 |

Related: #5022

**Comment by dakahn** [comment]
@carbon-design-system/design didn't we address the contrast disparities on various backgrounds with placeholders last year? I thought I remembered us updating those tokens.

**Comment by laurenmrice** [comment]
The placeholder text color contrast was addressed in this [PR](https://github.com/carbon-design-system/carbon/pull/4799).

If the "placeholder" text is actually displaying the format of the input or a prompt then it needs to pass color contrast (ie the date picker component, which is `text-05`). If however it is non-informational text (ie an example) then it can remain the low contrast placeholder color `text-03`. More information on this here: https://github.com/carbon-design-system/carbon/pull/4799#pullrequestreview-333428266

**Comment by dakahn** [comment]
Sounds good. closing as resolved -- happy to reopen if that's not the case.
